### PR TITLE
Update to Style System

### PR DESCRIPTION
**Pull request by alexasselin008**
- Semantic tokens for Dimensions are not added yet, i'll create a PR in the future
- The latest token update has not been synced with this branch. I'll do it in another PR

### CODE REVIEW

File: docs/features/style-props/StyleProps.stories.mdx [hunk 1]
```diff
@@ -25,9 +25,9 @@ To apply a style value, set a value to any [style property](#properties) of an O
     paddingTop={3}
     paddingBottom={3}
     marginBottom={3}
-    color="alias-static-white"
+    color="#FFFFFF"
```
Comment (alexasselin008) [inline docs/features/style-props/StyleProps.stories.mdx:28]: samoyed

File: packages/components/src/layout/docs/Grid.stories.mdx [hunk 2]
```diff
@@ -204,13 +204,13 @@ A custom `minmax` function is available to support Orbiter spacing scalue values
     <Story name="fit-content function">
         <Grid
             templateColumns={[fitContent(9), fitContent(9), "1fr"]}
-            gap={2}
+            gap={80}
             alignItems="center"
-            color="alias-static-white"
+            color="#FFFFFF"
```
Comment (alexasselin008) [inline packages/components/src/layout/docs/Grid.stories.mdx:209]: samoyed

File: packages/components/src/layout/src/Inline.tsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ export function InnerInline({
     as = DefaultElement,
     children,
     forwardedRef,
-    gap = 5,
+    gap = 240,
```
Comment (alexasselin008) [inline packages/components/src/layout/src/Inline.tsx:72]: "1.25rem"

File: packages/components/src/layout/src/Stack.tsx [hunk 4]
```diff
@@ -69,7 +69,7 @@ export function InnerStack({
     as = DefaultElement,
     children,
     forwardedRef,
-    gap = 5,
+    gap = 240,
```
Comment (alexasselin008) [inline packages/components/src/layout/src/Stack.tsx:72]: "1.25rem"

File: packages/components/src/styling/src/styled-system/props-handler.ts [hunk 5]
```diff
@@ -0,0 +1,1672 @@
...
+    "danger-strong": "danger-surface-strong",
+    "danger-hover": "danger-surface-hover",
+    "danger-disabled": "danger-surface-disabled",
+    "upsell-active": "upsell-surface-active",
+    "upsell-disabled": "upsell-surface-disabled",
+    "upsell-weak": "upsell-surface-weak",
+    "upsell-hover": "upsell-surface-hover"
+} as const;
+
+const BorderWidthAndStyle = "1px solid";
```
Comment (fraincs) [inline packages/components/src/styling/src/styled-system/props-handler.ts:270]: Should this be a rem?

Comment (alexasselin008) [reply packages/components/src/styling/src/styled-system/props-handler.ts:270]: I think it's one of the few usecase where i'd want an hardcoded border? i never want the border to be less than a pixel

**Review by fraincs — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**